data-structures/linked-lists/node.py

```diff
@@ -0,0 +1,7 @@
+class Node:
```

Comment:
Make sure you undertstand what Node is doing and that you should get a string back as result from your code.